third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
> I am not familiar with all that's going on here, so take my opinion lightly. It's already uncomfortable that we must maintain separate build files that aren't upstreamed.

As am I. I have no other solution for this ATM. The alternative is to wait for a mingw package for libvterm - see https://github.com/Alexpux/MINGW-packages/pull/1605